elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -45,47 +43,51 @@ $nav-variables: (
   logo: (
     MinWidth: 135px
   )
-);
-$variables: map-merge($variables, $nav-variables);
+));
+
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
```

Comment:
what's the difference between this map and the first one? can they be combined?

Reply:
I'll add comments but you can't use the pfe-local function inside the same map that the value is first defined - so any internal map references like `Padding` mapping to `PaddingTop PaddingRight PaddingBottom PaddingLeft` need to be done in a 2nd map.